### ISSUE TITLE
fatal: unrecognised output format 'elf64' on OS X

### ISSUE THREAD

**Issue by chrisccerami** [phil-opp/blog_os#46]
```
➜  os-rust  nasm -f elf64 multiboot_header.asm
nasm: fatal: unrecognised output format 'elf64' - use -hf for a list
```

This is the error I get when going through part one of your tutorial when trying to specify elf64 as the format for the linked executable.

nasm -hf lists:
```
valid output formats for -f are (`*' denotes default):
  * bin       flat-form binary files (e.g. DOS .COM, .SYS)
    aout      Linux a.out object files
    aoutb     NetBSD/FreeBSD a.out object files
    coff      COFF (i386) object files (e.g. DJGPP for DOS)
    elf       ELF32 (i386) object files (e.g. Linux)
    as86      Linux as86 (bin86 version 0.3) object files
    obj       MS-DOS 16-bit/32-bit OMF object files
    win32     Microsoft Win32 (i386) object files
    rdf       Relocatable Dynamic Object File Format v2.0
    ieee      IEEE-695 (LADsoft variant) object file format
    macho     NeXTstep/OpenStep/Rhapsody/Darwin/MacOS X object files
```

**Comment by phil-opp** [comment]
What version of `nasm` do you have? Maybe `brew install nasm` helps.

**Comment by phil-opp** [comment]
Closing in favor of #55

**Comment by ggarek** [comment]
_on macOS 10.12.5_

Had same issue using nasm 0.98.40 _(was already installed)_, fixed after upgrading nasm via homebrew to 2.13.01

**Comment by brihijoshi** [comment]
Thanks @ggarek ! Had issues as I didn't restart the terminal session
